packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -442,16 +442,22 @@ Template data:
     ];
     expect(out.slice(0, 3)).toEqual(explanationInfo);
 
-    const mnemonic = out[3];
+    const titleRows = [out[3], out[7]];
```

Comment:
Mhh, could we make this more readable? Something along thse lines:
```js
###########
#${menmonic.padStart(n).padEnd(n)}#
###########
```

Reply:
I don't see how using `padStart` or `padEnd` will help with readability as it needs the whole length as an argument so you can't really chain them. You would have to calculate the proper length anyway, I'd say even in a more complex way than just doing `/2`.

Reply:
Yeah, true. Maybe something like: 

```
const space = ' '.repeat(n)
const line = `#${space.concat(mnemonic).concat(space)}#`
```

Reply:
That's pretty much the same thing, you're just using `concat` instead of variable interpolation.